### PR TITLE
Search or Build your own plugin

### PR DESCRIPTION
**Pull request by branflake2267**
I'd like to suggest adding more to help folks find more plugins and if not build it.

### CODE REVIEW

**Comment by branflake2267** [comment]
Running xcode is failing now in then?

File: README.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ about plugins, and how to use them, see
 These plugins are also available on
 [pub](https://pub.dartlang.org/flutter/plugins).
 
+## Search for plugins
```
Comment (goderbauer) [inline README.md]: Seems like this paragraph is just duplication of the previous one?

File: README.md [hunk 2]
```diff
@@ -13,6 +13,15 @@ about plugins, and how to use them, see
...
+
+The plugin list below lists some of the plugins available, try searching the [pub flutter plugins](https://pub.dartlang.org/flutter/plugins) for more native platform plugins.
+
+## Build your own plugin
```
Comment (goderbauer) [inline README.md]: +1 to putting that here.

**Comment by goderbauer** [comment]
Failure is likely just a flake. I restarted Travis.

**Pull request closed without merging**